src/main/java/net/pms/external/update/AutoUpdaterServerProperties.java

```diff
@@ -82,6 +82,14 @@ private String getPlatformSpecificKey(String key) {
 		String os = operatingSystem.toString();
 		if (os.startsWith("windows")) {
 			os = operatingSystem.getPlatformName();
```

Comment:
This line and your new code seems unnecessary because "operatingSystem.toString()" returns either "windows-x86" or "windows-x86_64". So the same names could have variables in "latest_version.properties", e.g. "DownloadUrl2.windows-x86" and "DownloadUrl2.windows-x86_64" and corresponding URL and file names.